ERCS/erc-7682.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: 7682
+title: Auxiliary Funds Capability
+description: A way for wallets to indicate to apps that they have access to additional funds
```

Comment:
```suggestion
description: An ERC-5792 capability allowing wallets to indicate that they have access to additional funds.
```

Your description was fine before, but I feel like explicitly mentioning 5792 in the description might be useful.